src/Module.php

```diff
@@ -80,6 +86,39 @@ public function onBeforeSendLivePreview(Event $event)
         }
     }
 
+    // define entry behaviors
+    // - custom index entry properties
+    public function onEntryDefineBehaviors(DefineBehaviorsEvent $event)
+    {
+        $entry = $event->sender;
+        if (
+            $entry->id &&
+            $entry->sectionId &&
+            strpos(
+                $entry->section->handle,
+                IndexEntryBehaviors::$sectionHandlePrefix,
+            ) === 0
+        ) {
+            $event->behaviors[$this->id . 'IndexEntry'] =
+                IndexEntryBehaviors::class;
+        }
+    }
+
+    // define custom index query
+    public function onEntryQueryDefineBehaviors(DefineBehaviorsEvent $event)
+    {
+        $event->behaviors[$this->id . EntryIndexQueryBehavior::class] =
+            EntryIndexQueryBehavior::class;
+    }
+
+    // define custom Section properties
+    public function onSectionDefineBehaviors(DefineBehaviorsEvent $event)
```

Comment:
I don’t think the `SectionRouterBehavior` class has been imported, or the `onSectionDefineBehaviors` property is being invoked below.